debezium-api/src/main/java/io/debezium/spi/topic/TopicNamingStrategy.java

```diff
@@ -35,6 +36,30 @@
 
     String transactionTopic();
 
+    /**
```

Comment:
@harveyyue I don't think these settings belong here

1) The comments clearly state that it is for regex only so it shold be pulled-down there
2) The class is named Topic*Naming*Strategy and these settings are completely unrelated to topic naming

Reply:
Move these settings to AbstractRegexTopicNamingStrategy.